jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
+
+    public static String createIDPayload(List<String> fieldNameList, List<String> fieldValueList){
```

Comment:
nit: the naming of this method is slightly misleading in that sounds like this method only creates payloads which contain and id property. 
Some requests/tests don't need to specify an ID, so perhaps a slightly more generic name.. e.g. `createPayload`

Reply:
I've changed this to the more generic createPayload now